cms/envs/aws.py

```diff
@@ -176,8 +176,14 @@
 ASSET_IGNORE_REGEX = ENV_TOKENS.get('ASSET_IGNORE_REGEX', ASSET_IGNORE_REGEX)
 
 # Theme overrides
+STATICFILES_STORAGE = 'openedx.core.djangoapps.theming.storage.ProductionStorage'
```

Comment:
We're explicitly overriding this here, because we now set a development default in `common.py`.